src/lib/y2storage/storage_env.rb

```diff
@@ -0,0 +1,86 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "singleton"
+
+module Y2Storage
+  # Class to read ENV values for storage
+  class StorageEnv
+    include Singleton
+    include Yast::Logger
+
+    ENV_MULTIPATH = "LIBSTORAGE_MULTIPATH_AUTOSTART".freeze
+
+    ENV_BIOS_RAID = "LIBSTORAGE_MDPART".freeze
```

Comment:
are you sure that linuxrc always export it upper cased? in ruby-bindings we did this: https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/debugger.rb#L105

Reply:
ah, see it below, so already done.

Reply:
Here we do the same, see `#read`.